File: examples/example1.pl
#!/usr/bin/perl

use warnings;
use strict;

use RNA;

my $seq1 = "CGCAGGGAUACCCGCG";

# compute minimum free energy (mfe) and corresponding structure
my ($ss, $mfe) = RNA::fold($seq1);

# print output
printf "%s [ %6.2f ]\n", $ss, $mfe;

File: examples/example1.py
import RNA

sequence = "CUCGUCGCCUUAAUCCAGUGCGGGCGCUAGACAUCUAGUUAUCGCCGCAA"

# compute minimum free energy (mfe) and corresponding structure
(structure, mfe) = RNA.fold(sequence)

# print output
print "%s\n%s [ %6.2f ]" % (structure, mfe)
